vars/autoDeployEnvironment.groovy

```diff
@@ -32,6 +32,11 @@ def call() {
       subscriptionName: subscription.qaName,
       aksSubscription: aksSubscriptions.perftest
     ],
+    dev: [
```

Comment:
is this just for testing? i expected you to just change preview to point to your new subscription?